lib/ruby_lsp/server.rb

```diff
@@ -296,7 +296,7 @@ def text_document_did_open(message)
       @mutex.synchronize do
         text_document = message.dig(:params, :textDocument)
         language_id = case text_document[:languageId]
-        when "erb"
+        when "erb", "eruby"
```

Comment:
Why doesn't neovim have the `erb` as a filetype tho?

Reply:
Looks like [this](https://github.com/neovim/neovim/blob/9768e88f3891b5594eb2d556bd3bdf40c61d46e1/runtime/lua/vim/filetype.lua#L434) is where this is defined, though I couldn’t say if there’s a reason for "eruby" over "erb".

Reply:
Normally, editors allow you to tell which language ID a file extension should be associated to.

That said, I have two comments about this:

1. I think it's fine to add `eruby` if NeoVim standardizes the `erb` extension as `eruby`
2. I think it's worth asking @adam12 if it's working properly for him since he added the configs. Adam, is the ERB support working properly for you? Do you need any additional changes or is shipping this PR the way to go?

Reply:
> I think it's fine to add eruby if NeoVim standardizes the erb extension as eruby
Vim/Neovim have both standardized on the `eruby` filetype for `.erb` files.

>  Adam, is the ERB support working properly for you?
This _was_ working for me. When I tested it and submitted the PR to `nvim-lspconfig`, it looked like it worked fine. No warnings, no errors, autocomplete/etc seemed fine.

Today I see the same issue tho.

After making the change, I went ahead and did a lot of anxillary tasks, which may have affected things. Updated Neovim, updated Ruby-LSP, etc.